generator/src/test/java/com/scottlogic/deg/generator/ConstraintBuilder.java

```diff
@@ -44,8 +43,6 @@ public ConstraintBuilder addNullConstraint(String fieldName){
     }
 
     private static Set<RuleInformation> rules(){
-        RuleDTO rule = new RuleDTO();
-        rule.rule = "rules";
-        return Collections.singleton(new RuleInformation(rule));
+        return Collections.singleton(new RuleInformation("rule"));
```

Comment:
The parameter here doesn't seem to gain us anything - what is the benefit over passing in null, or providing a parameterless constructor that behaves as if null has been passed in?

Reply:
no reason, just less changes.

also, i don't know if null is safe in this class, so i wouldn't want to introduce a way to add potentially breaking constructors

Reply:
The current constructor has logic that replaces nulls with the constant "Unnamed rule".  I get the "less changes" argument, but I don't like to see us passing in these mystery strings that don't really serve a purpose - this would be a good opportunity to remove them.